abdulfatai360/politico
# #163649469 Implement CRUD on Parties Endpoints

#### What does this PR do?
Build RESt API to handle CRUD on parties endpoints

#### Description of Task to be completed?
Have the following endpoints working:
```POST /api/v1/parties```
```GET /api/v1/parties```
```GET /api/v1/parties/<party-id>```
```PATCH /api/v1/parties/<party-id>/<party-name>```
```DELETE /api/v1/parties/<party-id>```

#### How should this be manually tested?
Clone this repo, ```cd``` into it
- On Postman
Run ```npm start```, then send your request using Postman
- On terminal
Run ```npm test```, all tests should pass

#### What are the relevant pivotal tracker stories?
 #163649469
